ecs-logs-collector.sh

```diff
@@ -403,6 +404,9 @@ get_containers_info()
 
     if [ -e /etc/ecs/ecs.config ]; then
       cp -f /etc/ecs/ecs.config ${info_system}/ecs-agent/ 2>&1
+      if grep --quiet "ECS_ENGINE_AUTH_DATA" ${info_system}/ecs-agent/ecs.config; then
+        sed -i 's/ECS_ENGINE_AUTH_DATA=.*/ECS_ENGINE_AUTH_DATA=/g' ${info_system}/ecs-agent/ecs.config
```

Comment:
It would be better to sanitize the data than remove it entirely, since this is an often mis-configured part.  What you have here is an improvement though, so I think we can address that concern separately.

Can you copy this up to the line with `docker inspect` as well?  Environment variables (including `ECS_ENGINE_AUTH_DATA` on the ECS agent container) will show up there as well.